### ISSUE TITLE
Update for iOS 14

### ISSUE THREAD

**Issue by roberkane212** [bschlenk/homebridge-roku#175]
I can still use my Roku tvs, however they don’t come up as tv’s they come yo as the home
![90045C7B-6B4F-44EB-981D-0AD60F233F47](https://user-images.githubusercontent.com/44589993/86072388-7537df00-ba4f-11ea-85fd-e3a90cfbe68c.png)

**Comment by prhiggins** [comment]
Installing developer beta 2 fixed this for me.

**Comment by roberkane212** [comment]
@prhiggins same for me

**Comment by zimmra** [comment]
This was re-introduced to me with the public iOS 14 drop this week, Roku's show up as 'Home' icons. Functionality doesn't seem to be affected.

**Comment by rcoletti116** [comment]
Yes, icon on ios14 is the house. I've read that some other plugins have had to make changes to declare the accessory specifically as a Television. 
`accessory.category = api.hap.Categories.TELEVISION`

Also, I thought this was an external accessory, but now Home app won't let me remove the roku as a single device. The discussions on discord say it has to be an external accessory, not bridged.

**Comment by LordZork** [comment]
Yes, I am running the latest official/public release of iOS 14, and I just installed this plugin for the first time.  The setup was easy and functionally it seems fine, but – in the Home app – it is showing with the home icon instead of the TV icon.

Note: before installing and using this plugin, I was using the [homebridge-roku-tv](https://github.com/JarekToro/homebridge-roku-tv) plugin, until my device (Roku TV) started showing as "unresponsive," without any update/fix (for months now), which required frequent restarting of Homebridge to resolve.  That said, with that plugin, and despite the aforementioned issue, my Roku TV – even after the iOS 14 update – still had the TV icon, within the Home app.  I am not sure if it is simply because that icon carried over with the update, or if it has anything to do with the plugin itself.

I am hoping a fix is possible and is coming soon!  Not a fan of devices showing the wrong icon!

**Comment by imtitanic** [comment]
any update for this? :(

**Comment by BigIron5** [comment]
> Yes, icon on ios14 is the house. I've read that some other plugins have had to make changes to declare the accessory specifically as a Television.
> `accessory.category = api.hap.Categories.TELEVISION`
> 
> Also, I thought this was an external accessory, but now Home app won't let me remove the roku as a single device. The discussions on discord say it has to be an external accessory, not bridged.

This is the issue I'm having, too. 

Is this something we can do ourselves, or do we have to wait for the developer to update this plugin?

**Comment by raygan** [comment]
I'm still hoping for an update to this. It seems that this will require the developer moving the TV device to an external accessory rather than a bridged accessory.

After testing the native HomeKit support that is now built into my TCL Roku TV, I've reverted back to this plugin. This plugin allows me to control volume via the Control Center remote (with a sound bar via connected via HDMI) while the native HomeKit support doesn't allow for volume control on my setup for some reason.
